### ISSUE TITLE
Document not scanned when uploaded from CIFS

### ISSUE THREAD

**Issue by gmerbeth** [Redpill-Linpro/alfresco-clamav#1]
Hello and thank you for this great work !

I installed it on our alfresco 5.0.d community, and it worked !! But few details would not, see below  ...
- when uploading documents via CIFS, the file is not scanned. (onCreateNode policy not fired maybe).
- updating the "Update cron expression" in Overview panel is not working
- Scan job page is empty (not really a problem)

Maybe you could help me resolve the first one about CIFS, which is the only valuable problem I could find.

**Comment by oakman** [comment]
I really haven't validated this for 5.0.x yet, but please create separate tickets for each issue and we'll look into it as soon as possible.

**Comment by gmerbeth** [comment]
Hi oakman, 
I changed the binded policy in the <i>VirusCheckerBehaviour.java</i> file.
Instead of <i>OnCreateNodePolicy</i>, I used <i>org.alfresco.repo.content.ContentServicePolicies.OnContentUpdatePolicy</i>.

It seems to work now (for the CIFS issue).

There are also a problem of authentification context for the jobs, but it is too technical for me. Hope you'll be able to solve this when you are upgrading to 5.*.

For now, I'm happy with this.
A la prochaine ^^
